Dockerfile-ubuntu-20.04

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:20.04 as base
```

Comment:
For other reviewers, this line is the only difference between this file and the 18.04 version.

Reply:
...thankfully!